src/main/java/pages/TestTypeDetailsPage.java

```diff
@@ -510,6 +507,8 @@ public void sendCertificateNumberAndSave(String certificateNumber) {
     public void sendCertificateNumber(String certificateNumber) {
         logger.info("Setting Certificate Number to " + certificateNumber);
```

Comment:
Please add proper description to the PR to indicate what is covered in these auto tests

Reply:
Updated

Reply:
I dont see any changes